app/assets/stylesheets/welcome.scss

```diff
@@ -0,0 +1,63 @@
+
+
+video {
+  position: fixed;
```

Comment:
Properties should be ordered background, background-size, height, left, margin, min-height, min-width, position, top, -moz-transform, -ms-transform, -webkit-transform, transform, width, z-index